src/utils.ts

```diff
@@ -369,3 +369,11 @@ export const getVersion = () => {
     DEFAULT_VERSION
   );
 };
+
+export const getHumanVersion = () => {
+  const version = getVersion();
+  if (version.length === 28) {
+    return version.substr(0, 16).replace("T", " ");
+  }
```

Comment:
This will fail because commit hash length increases over time. I don't think it's necessary to do the check since we can automatically assume it does contain the ISO timestamp already.

Reply:
Btw, let's use `.slice()`. It's more powerful, more widely used, and `substr` is [considered legacy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr). It's also confusing (there are `.substr()` and `.substring()` with different behavior, and remembering the various rules between the 3 methods is hard enough).